pkg/appconsts/global_consts.go

```diff
@@ -13,16 +14,16 @@ import (
 // They can not change throughout the lifetime of a network.
 const (
 	// NamespaceVersionSize is the size of a namespace version in bytes.
-	NamespaceVersionSize = 1
+	NamespaceVersionSize = ns.NamespaceVersionSize
 	// NamespaceVersionMaxValue is the maximum value a namespace version can be.
 	// This const must be updated if NamespaceVersionSize is changed.
 	NamespaceVersionMaxValue = math.MaxUint8
```

Comment:
[extremely optional] It seems like this constant could also be exported by `pkg/namespace` and imported here

```suggestion
	// NamespaceVersionMaxValue is the maximum value a namespace version can be.
	// This const must be updated if NamespaceVersionSize is changed.
	NamespaceVersionMaxValue = ns.NamespaceVersionMaxValue
```